coalib/coala_modes.py

```diff
@@ -26,7 +26,8 @@ def mode_normal(console_printer, log_printer, args, debug=False):
         console_printer=console_printer,
         args=args,
         debug=debug)
-
+    if log_printer:
```

Comment:
Does this change really belong to this issue?
Commits should be atomic and no one commit should not deal with multiple changes.

Reply:
no ! mistakenly i do that actually i was working in same branch for two different issue h

Reply:
how to delete that commit ??

Reply:
Both the changes are in one commit only. So you could try to make relevant changes to the file (remove those extra lines) and save the file. After you are done with it, do `git add <filename>`. Then `git commit --amend` and save it. Then push it with `-f` option.

Reply:
Also when you do `git --amend` you will get an option to change the commit message. You should make the changes that I suggested (for the commit message) then.